src/healthcareapis/azext_healthcareapis/manual/custom.py

```diff
@@ -0,0 +1,262 @@
+                                resource_group_name,
+                                resource_name):
+    return client.get(resource_group_name=resource_group_name,
+                      resource_name=resource_name)
+
+
+def healthcareapis_service_create(client,
+                                  resource_group_name,
+                                  resource_name,
+                                  kind,
+                                  location,
+                                  tags=None,
+                                  etag=None,
+                                  identity_type=None,
+                                  access_policies=None,
+                                  cosmos_db_configuration=None,
+                                  authentication_configuration=None,
+                                  cors_configuration=None,
+                                  private_endpoint_connections=None,
+                                  public_network_access=None,
+                                  export_configuration_storage_account_name=None,
+                                  no_wait=False):
+
+    export_configuration = {
+        'storage_account_name': export_configuration_storage_account_name
+    }
+
+    service_identity_type = {
+        'principal_id': None,
+        'tenant_id': None,
+        'type': identity_type,
+    }
+
+    properties = {
+        'access_policies': access_policies,
+        'cosmos_db_configuration': cosmos_db_configuration,
+        'authentication_configuration': authentication_configuration,
+        'cors_configuration': cors_configuration,
+        'private_endpoint_connections': private_endpoint_connections,
+        'public_network_access': public_network_access,
+        'export_configuration': export_configuration
+    }
+
+    service_description = {
+        'name': resource_name,
+        'kind': kind,
+        'location': location,
+        'tags': tags,
+        'etag': etag,
+        'identity': service_identity_type,
+        'properties': properties
+    }
+
+    print(service_description)
+
+    return sdk_no_wait(no_wait,
+                       client.create_or_update,
+                       resource_group_name=resource_group_name,
+                       resource_name=resource_name,
+                       service_description=service_description)
+
+
+def healthcareapis_service_update(client,
+                                  resource_group_name,
+                                  resource_name,
+                                  tags=None,
+                                  public_network_access=None,
+                                  no_wait=False):
+
+    return sdk_no_wait(no_wait,
+                       client.update,
+                       resource_group_name=resource_group_name,
+                       resource_name=resource_name,
+                       tags=tags,
+                       public_network_access=public_network_access)
```

Comment:
Is this function directly copied from the generated `custom.py`? You only need to put the functions that you need to modify in this manual `custom.py`. The 2 files will be merged and the function in the manual folder will overwrite the same-name function in generated folder.

Reply:
Removed all methods that weren't changed